packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
The same is true for Jest and I'd say Facebook in general. Let's see how this goes for node and then learn from whatever they do :)